### PR TITLE
Improve CARTO module coverage

### PR DESCRIPTION
**Pull request by alasarr**
Improve CARTO module coverage

### CODE REVIEW

File: test/modules/carto/carto-bqtiler-layer.spec.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ test('CartoBQTilerLayer', t => {
 });
 
 test('CartoBQTilerLayer#updateTileJSON', t => {
-  const fetch = mockFetchWithTileJSON();
+  const fetch = mockFetchMapsV2();
```
Comment (AdriSolid) [inline test/modules/carto/carto-bqtiler-layer.spec.js:18]: Shall we rename this const for something different than `fetch`? It sounds like the `window.fetch` method.

File: test/modules/carto/carto-sql-layer.spec.js [hunk 2]
```diff
@@ -59,3 +62,85 @@ test('CartoSQLLayer#updateTileJSON', t => {
...
+
+  const onAfterUpdate = ({layer, subLayers, subLayer}) => {
+    const {tilejson} = layer.state;
+    if (!tilejson) {
+      t.is(subLayers.length, 0, 'should no render subLayers');
+    } else {
+      t.is(subLayers.length, 1, 'should render a subLayer');
+      t.ok(Array.isArray(tilejson.tiles), 'tiles should be an array');
+      t.ok(subLayer instanceof MVTLayer, 'subLayer should be an MVT layer ');
+      t.is(subLayer.props.data, tilejson, 'data should be a tileJSON');
```
Comment (AdriSolid) [inline test/modules/carto/carto-sql-layer.spec.js:80]: Typo `'an'`
`Sublayer should be a MVT layer.`

**Review by AdriSolid — CHANGES_REQUESTED** [review]

File: test/modules/carto/carto-bqtiler-layer.spec.js (hunk 1 above)
Comment (alasarr) [reply test/modules/carto/carto-bqtiler-layer.spec.js:18]: Why? We're mocking global fetch method

Comment (AdriSolid) [reply test/modules/carto/carto-bqtiler-layer.spec.js:18]: Just suggesting not calling const names like `window` methods. It's okey anyway!

**Review by AdriSolid — APPROVED** [review]

**Comment by alasarr** [comment]
I need to figure out why is coverage decreasing even when we're just adding tests...

File: test/modules/carto/carto-bqtiler-layer.spec.js (hunk 1 above)
Comment (alasarr) [reply test/modules/carto/carto-bqtiler-layer.spec.js:18]: Ok, makes sense

**Comment by Pessimistress** [comment]
@alasarr The coverage number fluctuates for no reason sometimes. It's actually going up (our low bound was 81.x)

**Pull request merged**